charts/incubator/nullserv/values.yaml

```diff
@@ -0,0 +1,56 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+controller:
+  # -- Number of pods to load balance between
+  replicas: 2
+
+image:
+  # -- image repository
+  repository: ghcr.io/k8s-at-home/nullserv
+  # -- image tag
+  tag: 1.3.0
+  # -- image pull policy
+  pullPolicy: IfNotPresent
+
+    # -- environment variables. See more environment variables in the [nullserv documentation](https://github.com/bmrzycki/nullserv/blob/master/README.md).
+# @default -- See below
+env:
+  # -- Set the container timezone
+  TZ: UTC
+
+# -- Configures service settings for the chart.
+# @default -- See values.yaml
+service:
+  main:
+    primary: true
+    type: NodePort
+    ports:
+      http:
+        port: 80
+  https:
+    enabled: true
+    type: NodePort
+    ports:
+      https:
+        enabled: true
+        port: 443
```

Comment:
Does this have to be a separate service, or can it also be a second port on the main service?

Reply:
It does not have to be a separate service, and I would prefer it be one service with on port, but I couldn't tell how to add another port to the main service in the common v3 library chart. I'll look more into that.

Reply:
Well, that was easy. Now it is one service with multiple ports.